packaging/docker/Dockerfile

```diff
@@ -7,14 +7,17 @@ ARG ARCH=amd64
 # This image contains preinstalled dependecies
 FROM netdata/builder:${ARCH} as builder
 
+# One of 'nightly' or 'stable'
+ARG RELEASE_CHANNEL=nightly
+
 ENV JUDY_VER 1.0.5
 
 # Copy source
 COPY . /opt/netdata.git
 WORKDIR /opt/netdata.git
 
 # Install from source
-RUN chmod +x netdata-installer.sh && ./netdata-installer.sh --dont-wait --dont-start-it
+RUN chmod +x netdata-installer.sh && ./netdata-installer.sh --dont-wait --dont-start-it $([ "$RELEASE_CHANNEL" = stable ] && echo --stable-channel)
```

Comment:
Nit: These `chmod +x` steps are unnecessary if we commit said scripts into Git with the execute bits set in the first place.

Reply:
Not sure why there were there. This PR doesn't touch that though. Should look into it in a fresh new PR.